Projects/UOContent/Gumps/ConfirmHouseResize.cs

```
@@ -3,170 +3,171 @@
-            AddHtmlLocalized(240, 250, 170, 20, 1011012, 0x7FFF); // CANCEL
-        }
+        mobile.CloseGump<ConfirmHouseResize>();
+    }
+
+    protected override void BuildLayout(ref StaticGumpBuilder builder) {
+        builder.SetNoClose();
+
+        builder.AddPage();
+
+        builder.AddBackground(0, 0, 420, 280, 0x13BE);
+        builder.AddImageTiled(10, 10, 400, 20, 0xA40);
+        builder.AddAlphaRegion(10, 10, 400, 20);
+        builder.AddHtmlLocalized(10, 10, 400, 20, 1060635, 0x7800); // <CENTER>WARNING</CENTER>
+        builder.AddImageTiled(10, 40, 400, 200, 0xA40);
+        builder.AddAlphaRegion(10, 40, 400, 200);
+
+        /* You are attempting to resize your house. You will be refunded the house's
+        value directly to your bank box. All items in the house will *remain behind*
+        and can be *freely picked up by anyone*. Once the house is demolished, however,
+        only this account will be able to place on the land for one hour. This *will*
+        circumvent the normal 7-day waiting period (if it applies to you). This action
+        will not un-condemn any other houses on your account. If you have other,
+        grandfathered houses, this action *WILL* condemn them. Are you sure you wish
+        to continue?*/
+        builder.AddHtmlLocalized(10, 40, 400, 200, 1080196, 0x7F00, false, true);
+
+        builder.AddImageTiled(10, 250, 400, 20, 0xA40);
+        builder.AddAlphaRegion(10, 250, 400, 20);
+        builder.AddButton(10, 250, 0xFA5, 0xFA7, 1);
+        builder.AddButton(210, 250, 0xFA5, 0xFA7, 0);
+        builder.AddHtmlLocalized(40, 250, 170, 20, 1011036, 0x7FFF);  // OKAY
+        builder.AddHtmlLocalized(240, 250, 170, 20, 1011012, 0x7FFF); // CANCEL
+    }
 
-        public override void OnResponse(NetState state, in RelayInfo info)
+    public override void OnResponse(NetState state, in RelayInfo info)
+    {
+        if (info.ButtonID == 1 && !_house.Deleted)
```

Let's simplify OnResponse with a fail fast.

```cs
if (info.ButtonID > 1 || _house.Deleted)
{
    return;
}
```